packages/react-components/src/icons/orcid-social.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+/* istanbul ignore file */
+
+const orcidSocialIcon = (
+  <svg
+    width={28}
+    height={28}
+    viewBox="0 0 28 28"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <title>Orcid Social</title>
```

Comment:
In that case
```suggestion
    <title>ORCID</title>
```
since we don't really need to distinguish them to the user :sweat_smile: